### PR TITLE
Allow vnet access fix

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
Fixed related to https://github.com/Azure/azure-cli/issues/9051  

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by MikhailTryakhov** [comment]
@dsgouda looks like there are 2 jobs (looks like) are failing because of js sdk generation. can u please help to figure out and can we escalate Js job fixing/removing from required escalation?

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Comment by MikhailTryakhov** [comment]
@dsgouda  done

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/virtualNetwork.json [hunk 1]
```diff
@@ -1150,11 +1150,11 @@
       "properties": {
         "allowVirtualNetworkAccess": {
           "type": "boolean",
-          "description": "Whether the VMs in the linked virtual network space would be able to access all the VMs in local Virtual network space."
+          "description": "Whether the VMs in the local virtual network space would be able to access the VMs in remote virtual network space."
         },
         "allowForwardedTraffic": {
           "type": "boolean",
-          "description": "Whether the forwarded traffic from the VMs in the remote virtual network will be allowed/disallowed."
+          "description": "Whether the forwarded traffic from the VMs in the local virtual network will be allowed/disallowed in remote virtual network."
```
Comment (mhyllander) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/virtualNetwork.json]: Is this correct? The portal notes that "forwarded traffic" does not originate in the local vnet, so "from the VMs" in the local vnet would appear to be wrong.

**Review by mhyllander — COMMENTED** [review]
Ref https://github.com/Azure/azure-cli/issues/9051.

**Pull request merged**